### ISSUE TITLE
Leveraging BOSH-manifest "overrides" API for director and/or product bosh manifest

### ISSUE THREAD

**Issue by aegershman** [pivotal-cf/om#515]
Hi!

**TL;DR:** any thoughts on om implementing features for the Ops Manager APIs for bosh director product manifest overrides?

background- I noticed the [Ops Manager 2.10.2 release notes](https://docs.pivotal.io/ops-manager/2-10/release-notes.html#2-10-2) mentions this line:

>
>[Feature]: You can use the /api/v0/staged/products/{product_guid}/overrides Ops Manager API endpoint to edit the update and features sections in the manifest for an Ops Manager tile while you are in Advanced Mode. For more information, see Provide a new list of overrides for the given Product in the Ops Manager API documentation.
>

The [API endpoint for providing product manifest overrides is documented here](https://docs.pivotal.io/platform/2-10/opsman-api/#tag/Advanced-Manifest-Configuration/paths/~1api~1v0~1staged~1director~1overrides/put). 

In addition, there's the [BOSH director manifest overrides API endpoint](https://docs.pivotal.io/platform/2-10/opsman-api/#tag/Advanced-Manifest-Configuration/paths/~1api~1v0~1staged~1director~1overrides/put) which I believe has been there for a while, but I've never looked into it before -- but I can _definitely_ think of benefits to providing overrides to product tile bosh manifests, hence this issue.

I didn't see anything for bosh manifest overrides in `om configure-director` or `om configure-product`, or a different command call pertaining to the `**/overrides` API endpoints-- is there a way this Ops Manager feature be incorporated into `om` so manifest property overrides could be managed as code? I haven't tested it, but I'd prefer not to have to manage it purely with `om curl` and tasks, but I could try when it comes up. My initial thought is that this could be managed similarly to how `vmextensions` work.

Thanks for any consideration for it. Apologies for any word salad here, I'm typing this at the end of a long day. Thanks for your time 👍

**Comment by jtarchie** [comment]
This is a good idea. We are discussing it.

It looks the first step might be doing individual `om` commands -- ie `om bosh-override-director`.
It is an _advance_ feature, which we'd like to separate from the structured functionality of `om configure-*`.
That way it is very intentional of what is happening.

We liked to implement, but it might take awhile till we get to it.
We'd accept a PR if needed sooner rather than later.

**Comment by jaristiz** [comment]
As no comments were made after the last post, I am going to close this one and reopen it if the conversation gains some traction.
